R/ggplot-theme.R

```diff
@@ -24,18 +24,17 @@
 #'  theme_correlaid() +
 #'  add_correlaid_logo()
 theme_correlaid <- function(base_size = 14,
+                            base_family = "Roboto",
                             base_line_size = base_size / 28,
                             base_rect_size = base_size / 28,
                             grid = "XY") {
   if (length(grid) != 1 || !grepl("none|X|Y|x|y", grid)) {
     stop('`grid` must be a string: "none" or any combination of "X", "Y", "x", and "y"')
   }
 
-  if (sum(grepl("^Roboto$", systemfonts::system_fonts()$family)) > 0) {
-    base_family <- "Roboto"
-  } else {
+  if (!sum(grepl(paste0("^", base_family, "$"), systemfonts::system_fonts()$family)) > 0) {
```

Comment:
```suggestion
  if (!(base_family %in% systemfonts::system_fonts()$family)) {
```

Reply:
I wrote it that way because regex matching was somehow faster than fixed string matching using `==`

```r
base_family <- "Roboto"
bench::mark(
  regex = sum(grepl(paste0("^", base_family, "$"), systemfonts::system_fonts()$family)) > 0,
  fixed = base_family %in% systemfonts::system_fonts()$family
)
```
```
#> # A tibble: 2 × 13
#>   expression      min   median `itr/sec` mem_alloc `gc/sec` n_itr  n_gc total_time result  memory
#>   <bch:expr> <bch:tm> <bch:tm>     <dbl> <bch:byt>    <dbl> <int> <dbl>   <bch:tm> <list>  <list>
#> 1 regex        3.96ms   4.05ms      240.     214KB     2.04   118     1      491ms <lgl [… <Rpro…
#> 2 fixed           4ms   5.17ms      187.     117KB     0       94     0      502ms <lgl [… <Rpro…
#> # … with 2 more variables: time <list>, gc <list>
```

But you're right, maybe readability is more important here than saving 1ms per plot? :D

Reply:
Ah interesting! I would not have anticipated that! 
Haha, yes unless it is intended to create 1 B of plots every day, simplicity probably > speed :D 
But as you prefer in the end, of course! :pray: